tests/integration/cattletest/core/test_authorization.py

```diff
@@ -172,14 +172,15 @@ def test_project_member_auth(admin_user_client, user_client, project_client):
 
 
 def test_host_auth(admin_user_client, user_client, project_client):
-    auth_check(admin_user_client.schema, 'host', 'r', {
+    auth_check(admin_user_client.schema, 'host', 'ru', {
         'accountId': 'r',
-        'apiProxy': 'r',
+        'apiProxy': ['r', 'u'],
```

Comment:
As a side effect this does work, but the intention was just to have the string 'ru'.  Can you change to 'ru'